index.js

```diff
@@ -121,6 +152,26 @@ function handleArguments(env) {
     exit(1);
   }
 
+  customizeConfig(['home', 'project', 'cwd'].map(getConfigFile));
```

Comment:
This code seems a little convoluted for what it is doing.  Maybe you can refactor to make it more clear.